src/main/scala/tech/sourced/gemini/ConnectedComponents.scala

```diff
@@ -76,7 +77,7 @@ abstract class ConnectedComponents(log: Slf4jLogger) {
         }
 
         val bucketsSize = buckets.size - prevBucketsSize
-        log.info(s"Fetched $hashtable, $bucketsSize buckets")
+        log.debug(s"Fetched $hashtable, $bucketsSize buckets")
```

Comment:
let's move changes of logging to separate PR. It has nothing in common with packaging.

Reply:
I prefer keeping it here as it allows to clearly see the situation before/after the packaging.

All the rest improvements except for logging are kept for the separate PR.

Reply:
sorry, which situation? Please help me understand how internal logging in the middle of the program allows to see results of packaging?

Reply:
Sure - it was added along with missing logging at the beginning of the program, so you could clearly see sbt/JVM start overhead. The rest still holds.

Reply:
I do understand the need for the logging at the beginning of the program and it's already in master: https://github.com/src-d/gemini/blob/master/src/main/scala/tech/sourced/gemini/QueryApp.scala#L65

If you feel so strongly about keeping it here, please let me re-review the code to check logging too.

Reply:
This was about having it consistently in all commands. 

Please review logging as well, it's conveniently structured in a separate commits that should simplify the process.